idflakies-maven-plugin/src/main/java/edu/illinois/cs/dt/tools/plugin/ReplayMojo.java

```diff
@@ -37,13 +36,13 @@ public void execute(final ProjectWrapper project) {
                 if (testRunResultTry.isSuccess()) {
                     Files.write(outputPath, new Gson().toJson(testRunResultTry.get()).getBytes());
                 } else {
-                    TestPluginUtil.project.error(testRunResultTry.failed().get());
+                    getLog().error(testRunResultTry.failed().get());
```

Comment:
Change to use Logger.getGlobal().log(Level.ERROR, ...)

Reply:
ERROR-> SEVERE